README.md

```diff
@@ -6,4 +6,27 @@
 
 Make company global travel and accommodation easy and convenient for the strong workforce of savvy members of staff, by leveraging the modern web.
 
----
+## Project top-level directory structure
+
+```
+public                          # serve static files used in the swagger documentation
+    ├── api-docs     
```

Comment:
update the folder structure to reflect the directory change:  `public/api-docs/swagger` --> `src/docs/swagger`